requirements/base.txt

```diff
@@ -16,6 +16,7 @@ redis==4.6.0
 Pillow==9.5.0
 icalendar==5.0.7
 dealer==2.1.0
+setuptools==58.2.0
```

Comment:
It looks like this is an error with `pyYAML`. We're using `5.4.1` and the latest is `6.0.1`. I took a look [at the changelog](https://github.com/yaml/pyyaml/blob/master/CHANGES) and the version after the one we're on [seems to fix this](https://github.com/yaml/pyyaml/pull/564). 

So I think the solution here is to upgrade `pyYAML` rather than pin setuptools to an older version.